docs/docs-content/vm-management/create-manage-vm/migrate-vm-to-different-node.md

```diff
@@ -71,7 +74,11 @@ Kubernetes resource.
 - Ensure `LiveMigrate` is set as the eviction strategy for all affected VMs. When the host is put in maintenance mode,
   this feature allows for a smooth and uninterrupted migration process.
 
-### Evacuate VMs in Palette
+### Instructions
```

Comment:
I know you've used this heading multiple places, but not sure I'm a fan because it reads very authoritarian. Perhaps `Steps` would give me nicer feels?